### PR TITLE
doc: Protocols release notes additions

### PR DESCRIPTION
**Pull request by KAGA164**
Updates to release notes for 1.6.0

Signed-off-by: Kamil Gawor <Kamil.Gawor@nordicsemi.no>

### CODE REVIEW

**Review by doki-nordic — APPROVED** [review]

**Review by maje-emb — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst [hunk 1]
```diff
@@ -15,7 +15,7 @@ The most relevant changes that are present on the master branch of the |NCS|, as
 Highlights
 **********
 
-There are no entries for this section yet.
+* Integrated Apple Find My supported for production.
```
Comment (kapi-no) [inline doc/nrf/releases/release-notes-latest.rst:18]: Please remove it. I just learned that highlights should not be contributed by us.

**Review by kapi-no — CHANGES_REQUESTED** [review]

**Review by PaalKastnes — APPROVED** [review]

File: doc/nrf/releases/release-notes-latest.rst (hunk 1 above)
Comment (b-gent) [reply doc/nrf/releases/release-notes-latest.rst:18]: yes, but this is actually one of the highlights approved by PMT so we can keep it here.

**Review by kapi-no — APPROVED** [review]

**Review by peknis — APPROVED** [review]

**Comment by KAGA164** [comment]
I've added additional information about fixing the shell_bt_nus sample.

File: doc/nrf/releases/release-notes-latest.rst [hunk 2]
```diff
@@ -161,6 +162,18 @@ nRF9160
...
+------------
+
+* Added:
+
+  * Support for serialization of the :ref:`zephyr:bt_gap` and the :ref:`zephyr:bluetooth_connection_mgmt` API.
+  * :ref:`ble_rpc_host` sample that enables support for serialization of the :ref:`zephyr:bt_gap` and the :ref:`zephyr:bluetooth_connection_mgmt`.
+
+* Updated:
+
+  * :ref:`shell_bt_nus` sample - Fixed an issue where shell transport did not display the initial shell prompt *uart:~$* on the remote terminal.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-latest.rst:175]:
```suggestion
  * :ref:`shell_bt_nus` sample - Fixed an issue where shell transport did not display the initial shell prompt ``uart:~$`` on the remote terminal.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-latest.rst (hunk 2 above)
Comment (KAGA164) [reply doc/nrf/releases/release-notes-latest.rst]: Done

**Review by peknis — APPROVED** [review]

**Comment by b-gent** [comment]
Done in https://github.com/nrfconnect/sdk-nrf/pull/4966

**Pull request closed without merging**